src/Cassandra.IntegrationTests/Policies/Tests/RetryPolicyTests.cs

```diff
@@ -26,6 +26,18 @@ namespace Cassandra.IntegrationTests.Policies.Tests
     [TestFixture, Category("long")]
     public class RetryPolicyTests : TestGlobals
     {
+        private SCassandraManager _scassandraManager;
```

Comment:
Can we move these `SCassandra`-based tests into another fixture with category `short`?